lib/project_types/script/layers/infrastructure/script_project_repository.rb

```diff
@@ -10,6 +10,21 @@ class ScriptProjectRepository
         SCRIPT_JSON_FILENAME = "script.json"
         MUTABLE_ENV_VALUES = %i(uuid)
 
+        def self.create_project_directory(ctx:, directory:)
+          ctx.mkdir_p(directory)
+          change_directory(ctx: ctx, directory: directory)
+        end
+
+        def self.delete_project_directory(ctx:, directory:)
+          parent_directory = Pathname(directory).parent
```

Comment:
Is this being "too clever"? Should I just pass in the `initial_directory` from `Application::CreateScript` here?